xml/System.Formats.Cbor/CborReaderState.xml

```diff
@@ -34,7 +34,8 @@
       </ReturnValue>
       <MemberValue>19</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Indicates that the next CBOR data item encodes a <see cref="T:System.Boolean" /> value (major type 7).</summary>
+        <remarks>To be added.</remarks>
```

Comment:
@carlossanlop Remarks on enum fields are invalid and (now) cause build warnings. Can you update DocsPortingTool to not add this for enum fields?